tf_big/cc/big_tensor.h

```diff
@@ -95,6 +106,30 @@ struct BigTensor {
       }
     }
   }
+  template <typename T>
+  void LimbsFromTensor(const Tensor& t) {
+    int rows = t.dim_size(0);
+    int cols = t.dim_size(1);
+    size_t num_real_limbs =
+        t.dim_size(2) * sizeof(T) - 4;  // get rid of header length
+
+    value = MatrixXm(rows, cols);
+
+    auto input_tensor = t.flat<T>();
+    const uint8_t* buffer =
+        reinterpret_cast<const uint8_t*>(input_tensor.data());
+
+    size_t pointer = 0;
+    for (int i = 0; i < rows; i++) {
+      for (int j = 0; j < cols; j++) {
+        unsigned int length = decode_length(buffer + pointer);
+        pointer += 4;
+        mpz_import(value(i, j).get_mpz_t(), length, 1, sizeof(uint8_t), 0, 0,
```

Comment:
will this work of int32_t as well? if not we should take it out of the supported dtypes for the op+kernel

Reply:
Yes. When I've tested the code locally I've done it using int32 as well, I should add those tests as well in the tensor test.

Reply:
Oh, now that I'm looking at it seems that the testing is done with int32. I don't see the problem then

Reply:
👍  merge at will!